interface/themes/core.scss

```diff
@@ -8,6 +8,13 @@
  * @license https://github.com/openemr/openemr/blob/master/LICENSE General Public License 3
  */
 
+// Globally enable gradients for themes on Bootstrap
+$enable-gradients: true;
+// Enables responsive text for Bootstrap (Will need to fix up)
+$enable-responsive-font-sizes: true;
+// Import Bootstrap 4 SCSS Files before doing anything
+@import "../../node_modules/bootstrap/scss/bootstrap";
```

Comment:
note that the node_modules directory does not exist in the production package (we delete it)

Reply:
This is a case of "if I don't write node_modules it fails to compile and if I write the public assets it does not" soooo I'm not sure how to correct that.

Reply:
Is the compiling happening only at build or is this something that needs to be done all the time?

Reply:
And if need to do all the time, are there any dependencies that are needed to support that?

Reply:
(btw, any of the above scenarios we should be able to solve quickly :) )

Reply:
I'm using docker to compile this at the moment and it snags there because it runs gulp before it moves the files to public assets which causes the SASS file to not be found.

Reply:
In the build, here's where gulp/sass is done which takes over end product to public/themes:
https://github.com/openemr/openemr/blob/master/gulpfile.js#L105

Reply:
Issue is while I'm really good with SASS, I'm terrible with gulp 😂

Reply:
Issue is while I'm really good with SASS, I'm terrible with gulp 😂

Reply:
hi @tywrenn ,
Just brought in a commit that will bring over the scss bootstrap directory over to public/assets/bootstrap on the build. Let me know if that works for you.

Reply:
That worked great! :)

Reply:
actually, it just hit me that this is only used on the build, so no need to bring this over to public/assets. Best to just do as you are doing and use from node_modules since this is during the build (removing of node_modules happens after the build). So, I'm gonna revert my recent commit.

Reply:
@tywrenn , Let me know if my logic in my post above is correct. I'm still confused whether the stuff in scss is only needed at build or if it is needed all the time when using OpenEMR (I can do gulp but not sass :) )

Reply:
If end users wish to build themes via sass, though, maybe should keep in public/assets. brain hurts :)

Reply:
(decision is to keep this for now since it works :) )

Reply:
Soooo are we keeping what I wrote or changing it? 😂😂

Reply:
keeping my change to bring in the scss into public assets, so should use the following instead:
@import "../../public/assets/bootstrap/scss/bootstrap";